app/src/main/res/layout/dialog_insert_character_id.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
made a **ctrl + l** in the code us not bad 😉

Reply:
Done!